File: Java/bubbleSort.java
public class bubbleSort {

    public static void main(String args[]){

        int arr[]= {7,2,1,23,18,9}; // example of an unsorted array

        //bubble sort code

        for(int i=0; i<arr.length-1; i++)
		{
			for(int j=0; j<arr.length-i-1;j++)
			{
				if(arr[j]>arr[j+1])
				{
					int temp=arr[j];
					arr[j]=arr[j+1];
					arr[j+1]=temp;
				}
			}
		}
    }
    
}

File: Python/bubbleSort.py
def bubbleSort(arr):
    n = len(arr)
    swapped = False
    for i in range(n-1):
        for j in range(0, n-i-1):
            if arr[j] > arr[j + 1]:
                swapped = True
                arr[j], arr[j + 1] = arr[j + 1], arr[j]

        if not swapped:
            return

arr = [25, 87, 60, 35, 75, 43, 22]

bubbleSort(arr)

print("Sorted array is:")
for i in range(len(arr)):
    print("% d" % arr[i], end=" ")
